test/DartTest.ts

```diff
@@ -306,6 +306,18 @@ describe('transpile to dart', () => {
     it('translates "void"', () => {
       chai.expect(() => translateSource('void x;')).to.throw('void operator is unsupported');
     });
+    it('translates "super()" constructor calls', () => {
+      expectTranslate('class X { constructor() { super(1); } }')
+          .to.equal(' class X { X ( ) : super ( 1 ) { /* super call moved to initializer */ ; } }');
+      expectTranslate('class X { constructor() { x(); if (y) super(1, 2); z(); } }')
+          .to.equal(' class X { X ( ) : super ( 1 , 2 ) { x ( ) ;' +
+                    ' if ( y ) /* super call moved to initializer */ ;' +
```

Comment:
That is a troubling change in behavior. Did you find an example of how the current transpiler handles this case?